lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2460,6 +2460,67 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      mme_pool_1:
+        type: string
+        format: ipv4
+        example: ''
+      mme_pool_2:
+        type: string
+        format: ipv4
+        example: ''
```

Comment:
I think some details that are currently unspecified can lead to some confusion:
- Are these optional?
- Is it possible to specify `mme_pool_1`, but leave `mme_pool_2` empty?

Please add some details here to make using the API more self-serviceable.

Reply:
You can probably add the `required: true` swagger tag at the places where these definitions are used in the API. Otherwise I think they are optional by default?